src/components/FlowPlayer/FlowPlayer.scss

```diff
@@ -0,0 +1,340 @@
+@use "sass:math";
+
```

Comment:
De styling gaan we idealiter toevoegen in de client zelf, enkel styling die nodig is om de component functioneel werkende te krijgen mag hier wel toegevoegd worden

Reply:
Dit moeten we best ff bespreken op standup. Want nu denk ik is in de inschatting enkel rekening gehouden met: we gebruiken de flowplayer van avo2, dus er is bijna geen werk aan.